make/photon/prepare/templates/docker_compose/docker-compose.yml.jinja

```diff
@@ -378,6 +378,9 @@ services:
     volumes:
       - {{data_volume}}/chart_storage:/chart_storage:z
       - ./common/config/chartserver:/etc/chartserver:z
+{% if gcs_keyfile %}
```

Comment:
Please do not take gcs_keyfile as a condition, this file is generated with empty content in harbor tile env, use storage_service is gcp